### DISCUSSION TITLE
completely disable noice

### DISCUSSION THREAD

**Discussion by 3ynm** [LazyVim/LazyVim#1462] [Q&A] [answered]
Is it possible to completely disable noice? I've been able to mostly disable it as [instructed on docs](https://www.lazyvim.org/configuration/plugins#-disabling-plugins), but I still see some messages popping ocassionally, for instance when there are updates or I make a config change.

**Comment by abeldekat** [comment, marked as answer]
Hello @hacktivista,

To avoid any popups, disable nvim-notify as well.

Do note that both noice and nvim-notify are not the source of the messages.

The messages for updates and config changes can be disabled in `lazy.lua`, according to `lazy.nvim`'s [configuration](https://github.com/folke/lazy.nvim#%EF%B8%8F-configuration). Look for the *checker* and *change_detection* properties.
